QueryCiv3/BiqSections/Bldg.cs

```diff
@@ -0,0 +1,126 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace QueryCiv3.Biq
+{
+    [StructLayout(LayoutKind.Sequential, Pack=1)]
+    public unsafe struct BLDG
+    {
+        public int Length;
+
+        private fixed byte Text[128];
+        public string Description { get => Util.GetString(ref this, 4, 64); }
+        public string Name { get => Util.GetString(ref this, 68, 32); }
+        public string CivilopediaEntry { get => Util.GetString(ref this, 100, 32); }
```

Comment:
Yep, the getters don't affect the memory layout, which is entirely determined by the ints and bytes and so on. We can't define flag fields or string fields because bit types don't exist and strings are variable length, so for those sections of memory, we can just define a buffer and use getters to parse out the data we need as necessary.

Reply:
Neat!  Score a point for C# as a cool language!